src/cocotb/_gpi_triggers.py

```diff
@@ -32,6 +32,9 @@
 if sys.version_info >= (3, 11):
     from typing import Self
 
+if sys.version_info >= (3, 10):
+    from typing import Literal  # noqa: F401  # used in type strings
```

Comment:
Christ. So ruff is incorrectly decided that it wasn't used and removed it because it was in a TypeAlias?